packages/web3/test/stress/large_request_data/index.js

```diff
@@ -0,0 +1,71 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/* eslint-disable */
+const { Web3 } = require('../../../lib/commonjs');
+const { IpcProvider } = require('../../../../web3-providers-ipc/lib/commonjs');
+const accounts = require('../../../../../scripts/accounts.json');
```

Comment:
convert tests to ts and import from web3 source as we are doing in other tests, instead of referring output dirs,